src/librustdoc/lib.rs

```diff
@@ -163,9 +163,6 @@ pub fn opts() -> Vec<RustcOptGroup> {
         stable("extern", |o| {
             o.optmulti("", "extern", "pass an --extern to rustc", "NAME=PATH")
         }),
-        stable("plugin-path", |o| {
-            o.optmulti("", "plugin-path", "directory to load plugins from", "DIR")
-        }),
```

Comment:
We could also keep the flags and have them do nothing; not sure what your opinion is!

Reply:
That's an interesting point. These flags are "stable", but enabled insecure code. While we can change stable things in the interest of security, i think it would be better to keep them and do nothing. At least then we can choose to link to this PR or the CVE instead of just reporting their absence.

Reply:
I think of something a bit different: keeping the option but printing that it's doing nothing.